### ISSUE TITLE
Usage of Travis CI

### ISSUE THREAD

**Issue by jgonsior** [Wikidata/QueryAnalysis#2]
We are already using JUnit and Maven, so I guess it should no problem to use an automatic deployment solution like Travis CI.

**Comment by mkroetzsch** [comment]
Not a problem, and generally a good thing if you have tests at all (setup is not s much work). However, since we are not developing software for others, quality requirements are "research prototype" level. Let's not spend too much time writing tests for things that are possibly used only once.

**Comment by jgonsior** [comment]
True, true, but I think if our script runs in the end for multiple hours it's not a bad idea either to have some tests which kind of ensure us that we aren't just wasting resources - so I guess we need to try to strike a balance between "over testing" and testing too less.
